index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/5be9dbaed1.js" crossorigin="anonymous"></script>
+    <title>Portfolio Samuel!</title>
+    
+</head>
+
+<body>
+
+ 
+    <header>
+      <a href="">Welcome</a>
+        <nav>
+           <button id="bmenu" type="button"><span class="material-symbols-outlined"> menu </span></button>
+        </nav>
+    </header>
+
+
+    <div id="main-container">
+
+
+    <div class="introduction">
+
+                    <h1>Hey there. I`m Samuel glad to see you</h1>
+                    <h2>I`m a sofware developer</h2>
+                    <p>I`m a student in the world of sofware developer, in a few months I be able
+                        to help to build a product, feature or website Look througth some of my work and
+                        experience! if you like what you see and have a project you need coded don`t hestiate to contact me.
+                    </p>
+                    
+                    <ul class="social-button">
+
+                    <li>
+                    <button class="social-t" type="button"><i class="fa-brands fa-twitter"></i></button>
+                  </li>
+                    <li>
+                    <button class="social-t" type="button"><i class="fa-brands fa-linkedin-in"></i></button>
+                  </li>
+                    <li>
+                    <button class="social-t" type="button"><i class="fa-brands fa-medium"></i></button>    
+                  </li>
+                  <li>
+                    <button class="social-t" type="button"><i class="fa-brands fa-github"></i></button>
+                  </li>
+                    <li>
+                    <button class="social-t" type="button"><i class="fa-brands fa-angellist"></i></button>
```

Comment:
- [ ] As suggested by the previous reviewer [here](https://github.com/SamuelTTW/Portfolio-Samuel-Parra-2/pull/1#discussion_r884674217), kindly use `a `tag instead of `button `.